be/src/column/column_helper.h

```diff
@@ -163,6 +163,11 @@ class ColumnHelper {
         if (src_column->is_nullable()) {
             return src_column;
         }
+
+        // const column should not be casted to nullable column.
+        if (src_column->is_constant()) {
+            return src_column;
+        }
```

Comment:
this func guarantee outputing an nullable column, i.e., a column is warped with nullableColumn. this lines break the guarantee.

for your case, the const column may be need to be unpack first, i.e., call `unpack_and_duplicate_const_column` first

Reply:
Since the `ColumnPtr ColumnHelper::create_const_null_column(size_t chunk_size` only needs the size of the column, I think it's no need to unpack the const column.